apcd-cms/src/client/src/hooks/cdls/useCDLs.ts

```diff
@@ -0,0 +1,21 @@
+import { useQuery, UseQueryResult } from 'react-query';
+import { fetchUtil } from 'utils/fetchUtil';
+import { cdlObject } from '.';
+
+const getCDLS = async (file_type: string | undefined) => {
+  const url = `common_api/cdls/${file_type}`;
+  const response = await fetchUtil({
+    url,
+  });
+  return response;
+};
+
+export const useCDLs = (
+  file_type: string | undefined
+): UseQueryResult<cdlObject> => {
+  const query = useQuery(['cdls', file_type], () =>
+    getCDLS(file_type)
+  ) as UseQueryResult<cdlObject>;
```

Comment:
Currently, it sends requests with undefined file_type. This will not send such requests.
```suggestion
{enabled: !!file_type}
  ) as UseQueryResult<cdlObject>;
```